apps/api/src/server/applications/application/__tests__/update-application.test.js

```diff
@@ -74,7 +74,7 @@ test('updates application with just title and first notified date', async (t) =>
 	const response = await request.patch('/applications/1').send({
 		title: 'some title',
 		keyDates: {
-			firstNotifiedDate: 123_456_789
+			submissionDateInternal: 1_649_319_344_000
```

Comment:
wait, are you not expecting a unix timestamp?

Reply:
a unix timestamp is a number isn't it?

Reply:
yep